README.md

```diff
@@ -8,8 +8,10 @@ Deequ is a library built on top of Apache Spark for defining "unit tests for dat
 
 ## Requirements and Installation
 
-__Deequ__ depends on Java 8 and Apache Spark 2.2, and is available via [maven central](http://mvnrepository.com/artifact/com.amazon.deequ/deequ). Add the latest release as a dependency to your project:
+__Deequ__ depends on Java 8 and Apache Spark 2.2. It only uses non-experimental APIs, which should work up to Spark 2.4.  
```

Comment:
It is true that we support Spark 2.3 and 2.4.

However, we extend some internal catalyst-based aggregation functions and have to apply some hacks to get this to work across different versions: https://github.com/awslabs/deequ/blob/master/src/main/scala/com/amazon/deequ/analyzers/catalyst/AttributeReferenceCreation.java#L73

So the comment about the non-experimental APIs is not true unfortunately.

Reply:
Ok, thanks for clarifying. I've updated the wording a bit to make the risks clearer.